src/main/resources/META-INF/mods.toml

```diff
@@ -19,20 +19,20 @@ modId="minecolonies" #mandatory
 # The version number of the mod - there's a few well known ${} variables useable here or just hardcode it
 version="${file.jarVersion}" #mandatory
  # A display name for the mod
-displayName="Minecolonies" #mandatory
+displayName="MineColonies" #mandatory
 # A URL to query for updates for this mod. See the JSON update specification <here>
 #updateJSONURL="http://myurl.me/" #optional
 # A URL for the "homepage" for this mod, displayed in the mod UI
-# displayURL="http://example.com/" #optional
+displayURL="https://minecolonies.com" #optional
 # A file name (in the root of the mod JAR) containing a logo for display
-# logoFile="structurize.png" #optional
+logoFile="minecolonies.png" #optional
```

Comment:
(if you're still having issues) this file is relative to the resources directory

Reply:
Ahh thanks

Reply:
![image](https://user-images.githubusercontent.com/63612548/104815511-a1902e80-57da-11eb-8758-92c1eb5fb54a.png)
Works thanks!